lib/workers/audit_trail_attribution_middleware.rb

```diff
@@ -0,0 +1,13 @@
+module Workers
+  class AuditTrailAttributionMiddleware
+    def initialize(_options = nil); end
+
+    AUDIT_USER_NAME = '(GOV.UK Apply User)'.freeze
```

Comment:
Maybe "(Automated process)"? It's not really a "user" that performs the action.

Reply:
done